tracker/telegram/bot.py

```diff
@@ -67,14 +69,26 @@ async def start_message(message: Message) -> None:
     )
 
 
+@dp.message(Command("notify_about_new_issues"))
+async def subscribe_to_issue_notifications(msg: Message):
+    try:
+        telegram_user = TelegramUser.objects.get(telegram_id=msg.from_user.id)
```

Comment:
Use `filter(...).first()` instead of get. That way you won't get an exception. Just `None` if it doesn't exist